src/controller/cve.controller/index.js

```diff
@@ -585,13 +580,13 @@ router.put('/cve/:id/reject',
   /*
   #swagger.tags = ['CVE Record']
   #swagger.operationId = 'cveCnaUpdateReject'
-  #swagger.summary = "Update a CVE Record rejection by ID (This endpoint is to be used by CNAs.)"
+  #swagger.summary = "Updates an existing CVE Record with a reject record for the specified ID (accessible to CNAs and Secretariat)"
   #swagger.description = "
         <h2>Access Control</h2>
-        <p>No roles needed to access the endpoint</p>
+        <p>User must belong to an organization with the <b>CNA</b> or <b>Secretariat</b> role</p>
         <h2>Expected Behavior</h2>
-        <p><b>Admin User:</b> Can submit CVE records for CVE IDs that are owned by the organization</p>
-        <p><b>Regular User:</b> Can submit CVE records for CVE IDs that are owned by the organization</p>"
+        <p><b>Admin User:</b> Updates a CVE record for records owned by their organization</p>
+        <p><b>Regular User:</b> Updates a CVE record for records owned by their organization</p>"
```

Comment:
Should these be 'Secretariat' user instead of 'Regular'? Also, if they're both the same it could be on one line